google/cloud/security/common/data_access/forseti.py

```diff
@@ -254,6 +257,33 @@ def __repr__(self):
                 self.name,
                 self.display_name)
 
+    if schema_number >= 2.0:
```

Comment:
Defining the class can potentially access the database with the assumption that such table exists or will be created otherwise. I'd rather get a 'undefined type error' somewhere then accidentally writing this to a database with an earlier schema. Hence the if is required.